velero/schedule/common-service-db/cs-db-br-script-cm.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 kind: ConfigMap
 metadata:
   name: cs-db-br-configmap
-  namespace: <cs-db namespace>
+  namespace: cs
```

Comment:
Hi @bluzarraga 
Do we need to update this back to `<cs-db namespace>`?

Reply:
yes, good catch!